tests/suites/test_suite_ecp.function

```diff
@@ -1198,6 +1207,36 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:HAVE_FIX_NEGATIVE */
+void fix_negative( data_t *N_bin, int c, int bits )
+{
+    mbedtls_mpi C, M, N;
+
+    mbedtls_mpi_init( &C );
+    mbedtls_mpi_init( &M );
+    mbedtls_mpi_init( &N );
+
+    /* C = - c * 2^bits */
+    TEST_EQUAL( 0, mbedtls_mpi_lset( &C, -c ) );
+    TEST_EQUAL( 0, mbedtls_mpi_shift_l( &C, bits ) );
+
+    TEST_EQUAL( 0, mbedtls_mpi_read_binary( &N, N_bin->x, N_bin->len ) );
+    TEST_EQUAL( 0, mbedtls_mpi_grow( &N, C.n ) );
+
+    /* M = - ( C - N ) */
```

Comment:
Minor: I think `M = N - C` would be clearer, but I guess that depends on how you look at it. Though as of this commit, "N - C" matches the description better ("N + c * 2^bits") as well the next line of code.

Reply:
`M = N - C` just reiterates the code. `M = - (C - N)` conveys that M is negative, and its absolute value is |C| - |N|.